### DISCUSSION TITLE
The output_fields seems not work in search

### DISCUSSION THREAD

**Discussion by AliscaCL** [milvus-io/milvus#16322] [Q&A and General discussion]
The schema included [id, sentence_id, sentence_embedding]
But while I am trying to do search as follows:

similar_result_schema_iter = collection.search(
    data = sentence_embedding, 
    anns_field = "sentence_embedding",
    param = {...} ,
    limit  = .. ,
    output_fields = ['sentence_id'] 
)

Only distance and id are returned by the search function as follows:

[(distance: .. , id: ... )] 

Where should I find the sentence_id?

**Comment by hensleyl** [comment]
`print(similar_result_schema_iter[0][0].entity.get('sentence_id'))`
